infrastructure/observability/otel-collector-values.yaml

```
@@ -1,7 +1,8 @@
 mode: deployment
 
 image:
-  repository: otel/opentelemetry-collector-contrib
+  repository: grafana/pyroscope-otel-collector
```

are you sure it exists? I cannot find it on docker hub and otel collector has status: `ImagePullBackOff`

---

Definitely, it doesn't exist 
```suggestion
  repository: otel/opentelemetry-collector-contrib
```

---

Yeah, it doesn't, my mistake